doc/source/tune/_tutorials/tune-advanced-tutorial.rst

```diff
@@ -145,6 +145,38 @@ thus just use the same ``Trainable`` for the replay run.
         scheduler=replay,
         stop={"training_iteration": 100})
 
+Trainable Class API with Population Based Training
+--------------------------------------------------
+
+While using the function API is sufficient for most PBT workflows, sometimes it is better to use the Trainable Class API.
+For example, if your training requires setting up an expensive reinforcement learning environment, we don't want to start a
+new trial on every single exploit. If we use the Trainable Class API, we can resue actors if our trainable implements ``reset_config``
+and we can avoid the expensive setup.
```

Comment:
IMO I think we can get rid of this section (Trainable Class API)

Reply:
Removed